.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: node_js
 node_js:
-  - 'stable'
-  - 'lts/*'
-  - 'lts/dubnium'
+  - "stable"
+  - "lts/*"
+  - "lts/dubnium"
+cache:
+  npm: false
```

Comment:
Why this?

Reply:
That is not necessary, initially I thought it was skipping the install script but it was not. Removed in last commit.